indigo_app/templates/place/works.html

```diff
@@ -5,7 +5,7 @@
 {% block view-id %}library-view{% endblock %}
 
 {% block content %}
-<div class="container mt-3 mb-5">
+<div class="container-fluid px-5 mt-3 mb-5">
```

Comment:
Once you've fixed the `.row > .col` thing below, the `px-5` can fall away.

Reply:
changed